commands.py

```diff
@@ -0,0 +1,102 @@
+from SchemaKeys import SchemaKeys
```

Comment:
I see you will add ICommand, all command classes and Factory in one file ? 
I recommend each class in one file. and all files in one folder calls "commands"

/commands 
     /command_factory.py
     / Icommand.py
    / create_command.py
   / delete_command.py